lineman/app/components/navbar/navbar.scss

```diff
@@ -22,8 +33,10 @@
 
 .lmo-navbar__item{
   display: inline-block;
-  margin: 0 0 0 2px;
-  height: $navbar-height - 4px;
+  margin: 0;
+  .navbar-user-options .lmo-navbar__btn {
```

Comment:
Please don't use unless you have to. I think it's ok for a .fa-bell type thing, but not for other BEM class names.

I'm trying to enforce that our CSS follows the BEM style. (Have a read of http://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/ if you like)

In this case just put the declaration directly under this one, rather than nested within it.